tests/js/setup-globals.js

```diff
@@ -101,7 +101,11 @@ wooCommercePackages.forEach( ( lib ) => {
 } );
 
 const config = require( '../../config/development.json' );
-window.wcAdminFeatures = config && config.features ? config.features : {};
+
+// Check if test is jsdom or node
```

Comment:
This check means this setup file can run without crashing on `node` environment or `jsdom` environment, later we can start using node environment on tests that don't need DOM to speed things up further